src/LogoFX.Client.Core.Platform/src/CommonProperties.cs

```diff
@@ -24,6 +31,7 @@ public static DependencyObject GetOwner(DependencyObject obj)
         /// </summary>
         /// <param name="obj"></param>
         /// <param name="value"></param>
+        [SuppressMessage("ReSharper", "UnusedMember.Global")]
```

Comment:
@glyad Please remove - no need to add dependency on Resharper here

Reply:
Same as above...